en/release_note/release_note_latest_ver.rst

```diff
@@ -22,62 +24,66 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
 * fixes a large number of critical bugs.
-* includes useful SQL extensions: JSON data type and CTE(Common Table Expressions) extensions.
-* includes many enhancements to index management: Online parallel index loading, Faster foreign key loading, Invisible Index.
-* includes huge scale of code refactoring and modernization.
-
-CUBRID 10.2 is **faster**. It optimizes auto-commit workloads, for instance, YCSB Workload A (140%: 10.1) and YCSB Workload B (130%: 10.1). For faster bulk data loading, CUBRID 10.2 has online parallel loader which shows x10 performance to CUBRID 10.1.
+* includes useful SQL extensions: Supporting RVC (Row Value Constructor) and various REGEXP functions.
+* includes code refactoring and modernization.
 
-CUBRID 10.2 is also **better**, bringing new features. CUBRID 10.2 supports built-in JSON data types, path, functions and operators. Common Table Expression is now allowed for INSERT, REPLACE, DELETE, UPDATE, CREATE TABLE AS SELECT as well as SELECT. \
-Index management is greatly improved. To load a secondary key is no more painful. It supports online index loading which allows other select and DMLs without loosing throughputs. It even provides parallel option to load an online index, which greatly speeds up index loading. \
-It also provides from x2 to x3 performance boost to load foreign keys. \
-Invisible index helps DBAs to test effect of removing an index without making a disruptive change.
+CUBRID 11.0 **improves security** by providing data encryption and packet encryption. This version prevents abnormal data leakage by supporting table-based TDE (Transparent Data Encryption) and packet encryption between the driver and server.
 
-CUBRID 10.2 is more **reliable** and **stable** too. Double Write Buffer protects against I/O partial writes and includes many optimizations to minimize performance degradation. It has fixed over 1300+ issues as CUBRID 10.2 and passed rigorous tests.
+CUBRID 11.0 is **faster**. This version supports hash scan and improves the performances by up to 10 times in join query that could not perform index scans. By supporting the cache of search query results through hints, data change is minimal, and the performance of the workload with complex queries is maximized.
 
-CUBRID 10.2 is built with devtoolset-8(GCC 8.2) of Linux and VS 2017 of Windows. CUBRID code is completely recompiled with C++ and new features implementation fully exploit the C++ extensions and STL.
+CUBRID 11.0 **improves administrator convenience** by providing new functions for administrators. This version supports syntax-based replication through hints on the HA environment, improving the replication time when deleting and updating a large amount of data. By separating the Java SP server from the DB server, the influence of the DB server is minimized from the start/stop of the Java SP server. In addition, the DDL audit function is provided so that DDL change can be tracked.
```

Comment:
You are better to change "syntax based" to "statement based".

Reply:
I fixed it.